crates/chain/src/tx_graph.rs

```diff
@@ -549,6 +549,32 @@ impl<A: Clone + Ord> TxGraph<A> {
         self.apply_update(update)
     }
 
+    /// Update the last seen unconfirmed time for all unanchored transactions.
+    ///
+    /// Returns the [`ChangeSet`] after applying any updates to `self`. Note that
+    /// [`TxGraph`] only keeps track of the latest `seen_at`.
```

Comment:
It'd be nice if the docs explain more about why you would want to use this method. Are there some docs about how "last seen" works around that we could link to?

One question that comes to mind is why isn't it good enough to just set the `last_seen` to the most recent block header time. An example, showing how to get the right time from `std::time` would be good.

Reply:
Pushed a docs only commit, let me know what you think.

@evanlinjin Is there a preference for keeping `into_tx_graph` as shown in c551dfba304464cf0814a9211b66d34e513f53b7 versus dropping the `seen_at` parameter in favor of the new `update_last_seen_unconfirmed`?

Also, passing in a chain tip is an interesting idea that could be its own method if we could demonstrate a use for it, but isn't `TxGraph` supposed to be agnostic to whether a tx is "canonical"?

Reply:
docs are :100:. Your docs explain nicely why we don't want to use the block header time.

Reply:
@ValuedMammal do you mean dropping the `seen_at` parameter from `into_tx_graph`? I think that will be nice thanks!